src/log4net/Core/LoggingEvent.cs

```diff
@@ -895,7 +895,7 @@ public string UserName
 			{
 				if (m_data.UserName == null  && this.m_cacheUpdatable) 
 				{
-#if (NETCF || SSCLI || NETSTANDARD1_3) // NETSTANDARD1_3 TODO requires platform-specific code
+#if (NETCF || SSCLI || NETSTANDARD) // NETSTANDARD TODO requires platform-specific code
```

Comment:
win32 is a valid target for netstandard too -- surely there's a better alternative?

Reply:
(for example: if the platform is !win32, we could try to invoke `whoami`)